recipes/nuraft/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+import os
+from conan import ConanFile
+from conans import CMake
+from conan.tools.build import cross_building
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
```

Comment:
This should be the test for the new generators and this can be moved to `test_v1_package/`